packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -1,43 +1,38 @@
 import template from '@babel/template';
 import * as t from '@babel/types';
-import traverse, { Scope, NodePath, Visitor } from '@babel/traverse';
+import traverse, { NodePath, Visitor } from '@babel/traverse';
 import { hash, unique } from '@compiled/utils';
 import { transformCss } from '@compiled/css';
 import isPropValid from '@emotion/is-prop-valid';
-
-import { PluginOptions, Tag } from '../types';
-
+import { Tag } from '../types';
 import { CSSOutput } from './css-builders';
 import { pickFunctionBody } from './ast';
+import { CompiledOpts, CompiledTemplateOpts, StyledOpts, StyledTemplateOpts } from './types';
+import { Metadata } from '../types';
```

Comment:
nit: Two `types` files confused me at first 😅? Can we do something about these 🤔?

Reply:
oops

any suggestions? main thing i wanted to improve was the size of types is getting a bit crazy atm

Reply:
Yeah they do. `{ CompiledOpts, CompiledTemplateOpts, StyledOpts, StyledTemplateOpts }` seems more like `ASTUtilsTypes` something. Maybe we can have something better naming than this.

Reply:
Are we making this change @madou? Otherwise this looks good.

Reply:
let's make the change in a follow up PR 👍 (will modify the APIs of above funcs too)